common/test/acceptance/pages/studio/library.py

```diff
@@ -278,4 +280,7 @@ def refresh_children(self):
         btn_selector = self._bounded_selector(".library-update-btn")
         refresh_button = self.q(css=btn_selector)
         refresh_button.click()
```

Comment:
@kelketek OK I think I got it now. From the [latest screenshot](https://build.testeng.edx.org/job/edx-platform-all-tests-pr/68/SHARD=3,TEST_SUITE=bok-choy,label_exp=jenkins-worker/artifact/test_root/log/acceptance.tests.studio.test_studio_library_container.StudioLibraryContainerTest.test_settings_overrides.png) it looks like the click on the "Update now..." button never happened.
Here's the code:
```
        btn_selector = self._bounded_selector(".library-update-btn")
        refresh_button = self.q(css=btn_selector)
        refresh_button.click()
```
The q method returns a query object, and the click method will click each item. However if the query result was an empty list, because the element was not yet there, then the click will not fail, it will iterate over nothing.

The line prior to the refresh_children method call does self.assertTrue(library_block.has_validation_warning), but there could still be a synchronization issue, there are 2 p.warning elements and the browser could have only gotten to rendering the first one before the click method call.

I will add another commit to make sure that the refresh button is visible before trying to click on it.